### ISSUE TITLE
[glance] Cirros image is out of date and hates metadata

### ISSUE THREAD

**Issue by claco** [rcbops/chef-cookbooks#919]
Reminder for future self. 0.3.1 Cirros image we upload hates metadata and takes longer than normal to finish booting sometimes. 0.3.2 seems to fix a lot of things. Nothing major, unless you're using this image first to test your setup. :-/

https://launchpad.net/cirros/+milestone/0.3.2

**Comment by mancdaz** [comment]
Yeah they add parsing of option 121 to udhcpd:

https://bugs.launchpad.net/cirros/+bug/1190372

Nice.

**Comment by odyssey4me** [comment]
Yeah, I hit that a while back. I found out about the 0.3.2 image through some of the mailing list discussions. I find it odd that they haven't gotten to the point of releasing the update image yet - I wonder what the hold-up is.

**Comment by mancdaz** [comment]
@odyssey4me you mean this?:

http://download.cirros-cloud.net/0.3.2/cirros-0.3.2-x86_64-disk.img

**Comment by odyssey4me** [comment]
@mancdaz :) No, I mean why is that download not showing here: https://launchpad.net/cirros/+download

The 0.3.0 release has a listing of all the images. From then on it's only the source that's showing for download.

**Comment by mancdaz** [comment]
_shrug_ I guess they decided to host all but the source on download.cirros-cloud.net from 0.31 onwards

**Comment by cloudnull** [comment]
retest
